media/css/mozorg/contribute.scss

```diff
@@ -98,9 +98,58 @@ $image-path: '/media/protocol/img';
             flex-wrap: wrap;
         }
 
+        // protocol override to avoid over-frequent word breaks
+        &.mzp-l-card-quarter .mzp-c-card {
+            @media #{$mq-lg} {
+                @include bidi((
+                    (float, left, right),
+                    (margin-left, 0, $spacing-xl),
+                    (margin-right, $spacing-xl, 0),
+                ));
+                width: calc(50% - (#{$spacing-xl} * 0.5));
+
+                &.mzp-c-card-extra-small,
+                &,
+                &.mzp-c-card-medium,
+                &.mzp-c-card-large {
+                    max-width: 100%;
+                }
+
+                &:nth-child(even) {
+                    margin-left: 0;
+                    margin-right: 0;
+                }
+            }
+            @media (min-width: 1100px) {
```

Comment:
@maureenlholland Could you use hyphens for the headlines of the boxes but not for the content? The design is not really suitable for long words in the headlines, but the German language often has longer words than English*. But I agree that it's not necessarily a better user experience to break more words than needed. So it seems okay to me to give the headlines a special treatment.

The german language does not allow word separation in arbitrary places. I am not convinced that one issue should be solved by replacing with another issue. The result may looks better but it's just wrong from the perspective of the language.

*) Unfortunately this is a recurring issue with Mozilla websites that they are designed with only English in mind, see for example https://github.com/mozilla/fxa/issues/14268 or https://github.com/mozilla/blurts-server/issues/2684 or https://github.com/mozilla/fx-private-relay/issues/2508. 🙁

Reply:
@cadeyrn thanks for the feedback! We're a little limited by browser support for hyphens so I think I need to keep the overflow-wrap break-word rule as a fallback, but happy to use hyphens specifically on the headings of the card.

+1 on this:

> It happens why too often and has probably to be solved on a higher level: it needs more awareness for internationalization within the design team, especially since such issues can also cause accessibility issues.